packages/block-editor/src/components/inserter/menu.js

```diff
@@ -67,6 +67,8 @@ function InserterMenu(
 	const [ patternFilter, setPatternFilter ] = useState( 'all' );
 	const [ selectedMediaCategory, setSelectedMediaCategory ] =
 		useState( null );
+	const isLargeViewport = useViewportMatch( 'large', '>=' );
```

Comment:
```suggestion
	const isLargeViewport = useViewportMatch( 'large' );
```

Nit: `>=` is [the default operator](https://github.com/WordPress/gutenberg/blob/46db6e1195c4f1cd4ae5e00b1c8b951177fc8634/packages/compose/src/hooks/use-viewport-match/index.js#L76).

Reply:
Good catch! Thank you!